### PR TITLE
chore: bump dependencies

### LINKED ISSUE #7483: Enable dm-raid module to support LVM mirror LVs
### Discussed in https://github.com/siderolabs/talos/discussions/7481

<div type='discussions-op-text'>

<sup>Originally posted by **eshikhov** July 15, 2023</sup>
Hi!

To provide PV we use https://github.com/metal-stack/csi-driver-lvm, when using storageclass csi-driver-lvm-linear everything works, but when using csi-driver-lvm-mirror it gives the error: 
```
"Failed to load existing device cache from /etc/lvm/cache/.cache
    Executing: /sbin/modprobe dm-raid
modprobe: module dm-raid not found in modules.dep
  /sbin/modprobe failed: 1
  raid1: Required device-mapper target(s) not detected in your kernel.
  Run `lvcreate --help' for more information."
```
Tried to add via KernelConfig.modules add dm-raid talos gives error:
 ```
 "user: warning: [2023-07-14T21:06:37.868957414Z]: [talos] controller failed {" component": "controller-runtime", "controller": "runtime.KernelModuleSpecController", "error": "error loading module \x5c"md-raid\x5c": module not found"}"
``` 
it is possible to load the dm-raid module ?
</div>

### PR DESCRIPTION
**Pull request by smira**
* go.mod dependencies
* Linux 6.1.39
* runc 1.1.8
* dm-raid kernel module

Fixes #7483

### CODE REVIEW

File: hack/modules-arm64.txt [hunk 1]
```diff
@@ -1,6 +1,14 @@
 modules.order
```
Comment (frezbo) [inline hack/modules-arm64.txt:1]: is the list same for both amd64 and arm64, sometimes they differ a bit

**Review by frezbo — APPROVED** [review]

File: hack/modules-arm64.txt (hunk 1 above)
Comment (smira) [reply hack/modules-arm64.txt:1]: these should be arch-independent I believe

**Comment by smira** [comment]
`dm-raid` loads fine:

```
[   56.811386] raid6: avx2x4   gen() 47098 MB/s
[   56.879373] raid6: avx2x2   gen() 39802 MB/s
[   56.947361] raid6: avx2x1   gen() 29255 MB/s
[   56.947655] raid6: using algorithm avx2x4 gen() 47098 MB/s
[   57.015347] raid6: .... xor() 6602 MB/s, rmw enabled
[   57.015686] raid6: using avx2x2 recovery algorithm
[   57.019473] async_tx: api initialized (async)
[   57.021890] xor: automatically using best checksumming function   avx       
[   57.042285] device-mapper: raid: Loading target version 1.15.1
```

**Pull request merged**